apps/vapi/RenderManager.cpp

```diff
@@ -0,0 +1,239 @@
+#include "RenderManager.h"
+
+#include <vapor/GUIStateParams.h>
+#include <vapor/AnimationParams.h>
+#include <vapor/TrackBall.h>
+#include <vapor/ControlExecutive.h>
+#include <vapor/ParamsMgr.h>
+#include <vapor/GLManager.h>
+#include <vapor/Framebuffer.h>
+
+#include "GLInclude.h"
+
+#define INCLUDE_DEPRECATED_LEGACY_VECTOR_MATH
+#include <vapor/LegacyVectorMath.h>
+
+
+using namespace VAPoR;
+
+RenderManager::RenderManager(ControlExec *ce) : _controlExec(ce) {}
+
+RenderManager::~RenderManager() { delete _glManager; }
```

Comment:
_glManager is not is not allocated in the constructor and  may not be initialized, no?

Reply:
Fixed, thanks.